src/abstractdataframe/abstractdataframe.jl

```diff
@@ -1330,16 +1368,20 @@ julia> vcat(d4, df1)
 
 """
 Base.vcat(dfs::AbstractDataFrame...;
-          cols::Union{Symbol, AbstractVector{Symbol}}=:setequal) =
+          cols::Union{Symbol, AbstractVector{Symbol},
+                      AbstractVector{<:AbstractString}}=:setequal) =
```

Comment:
This will allow mixing strings and `Symbol`s. I leave it out for now as I want to hear what @nalimilan thinks about it (I will make a note in the PR to consider this).